css-grid-classes/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Grid Classes</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <header><h1>Choose your character!</h1></header>
+      </div>
+      <div class="row">
+        <div>
```

Comment:
Make sure to update each of the `row` direct children with their proper column classes